packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -83,20 +111,24 @@ export function ConnectionBootstrap({
     [connection]
   );
 
-  if (connection == null || error != null) {
-    return (
-      <LoadingOverlay
-        data-testid="connection-bootstrap-loading"
-        isLoading={connection == null}
-        errorMessage={error != null ? `${error}` : undefined}
-      />
-    );
```

Comment:
This shouldn't be removed

Reply:
If I don't remove this, then we cannot see the Authentication failed modal, since the overlay goes over it.